### PR TITLE
FSET-682: Download candidate adjustments report

### PR DESCRIPTION
**Pull request by Lansalo**
This should be merged along with https://github.tools.tax.service.gov.uk/HMRC/fset-faststream-admin-frontend/pull/38

### CODE REVIEW

File: app/model/Commands.scala [hunk 1]
```diff
@@ -62,8 +62,20 @@ object Commands {
     hasAdjustments: Option[String], guaranteedInterview: Option[String], firstName: Option[String], lastName: Option[String],
     preferredName: Option[String], dateOfBirth: Option[String], cubiksUserId: Option[Int])
 
-  case class AdjustmentReport(userId: String, firstName: Option[String], lastName: Option[String], preferredName: Option[String],
-    email: Option[String], telephone: Option[String], adjustments: Option[String], gis: Option[String], adjustmentsConfirmed: Option[String])
+  final case class AdjustmentReport(
```
Comment (mgilgar) [inline app/model/Commands.scala]: We have created a new folder called model.report where you can put this class and related stuff in a separate file. Maybe you have to merge from master in to your branch to see what I mean

File: app/repositories/application/GeneralApplicationRepository.scala [hunk 2]
```diff
@@ -1051,7 +1069,7 @@ class GeneralApplicationMongoRepository(timeZoneService: TimeZoneService)(implic
     val firstName = personalDetails.getAs[String]("firstName").get
     val lastName = personalDetails.getAs[String]("lastName").get
     val assistanceDetails = doc.getAs[BSONDocument]("assistance-details").get
-    val needsAdjustment = assistanceDetails.getAs[String]("needsAdjustment").get
+    val needsAdjustment = assistanceDetails.getAs[Boolean]("needsSupportAtVenue").flatMap(b => Some(booleanTranslator(b))).get
```
Comment (mgilgar) [inline app/repositories/application/GeneralApplicationRepository.scala]: I recommend to rename the name of the val to needsSupportAtVenue

File: app/model/Commands.scala (hunk 1 above)
Comment (Lansalo) [inline app/model/Commands.scala]: Done

File: app/repositories/application/GeneralApplicationRepository.scala (hunk 2 above)
Comment (Lansalo) [inline app/repositories/application/GeneralApplicationRepository.scala]: Done

**Pull request merged**